build/wikibase/entrypoint.sh

```diff
@@ -91,12 +91,10 @@ else
     php /var/www/html/maintenance/update.php --quick
 
     if [ -f /default-extra-install.sh ]; then
-        # shellcheck disable=SC1091
```

Comment:
why is this going away?

Reply:
Our spellcheck installation can now
find the context and so sees this script